qa/restart-upgrade/src/test/java/org/opensearch/neuralsearch/bwc/HybridSearchIT.java

```diff
@@ -69,6 +70,7 @@ private void validateNormalizationProcessor(final String fileName, final String
                 loadModel(modelId);
                 addDocuments(getIndexNameForTest(), false);
                 validateTestIndex(modelId, getIndexNameForTest(), searchPipelineName);
+                validateTestIndex(modelId, getIndexNameForTest(), searchPipelineName, Map.of("ef_search", 100));
```

Comment:
Instead of adding this validation in next line, we should pass this Map in line 72. Otherwise, it will repeat the same validation again. Instead of creating a copy method validateTestIndex ,Can we add extra parameter in the same method definition?

Reply:
> Instead of adding this validation in next line, we should pass this Map in line 72. Otherwise, it will repeat the same validation again

The intent is to do the validation again, with and without method parameters. so its called twice, The results should be the same

> Instead of creating a copy method validateTestIndex ,Can we add extra parameter in the same method definition?

Its not a copy method, its an overloaded method. The logic is being reused see lines 101 and 105